jenkins/ompi/ompi_test.sh

```diff
@@ -1,105 +1,98 @@
 #!/bin/bash -eEl
 
-# TODO:
-#   1. Check that yalla is not built
+set -u
+set -o pipefail
 
-if [ "$DEBUG" = "true" ]; then
+if [ "$DEBUG" = "true" ]
+then
     set -x
 fi
 
 # Check that you are inside a docker container
 cat /proc/1/cgroup
 
-# prepare to run from command line w/o jenkins
-if [ -z "$WORKSPACE" ]; then
+if [ -z "$WORKSPACE" ]
+then
     echo "WARNING: WORKSPACE is not defined"
-    WORKSPACE=$PWD
-    JOB_URL=$WORKSPACE
-    BUILD_NUMBER=1
-    JENKINS_RUN_TESTS=yes
-    NOJENKINS=${NOJENKINS:="yes"}
-    ghprbTargetBranch=${ghprbTargetBranch:="mellanox-v1.8"}
+    WORKSPACE="$PWD"
 fi
 
-cd $WORKSPACE
-
-export PATH=/hpc/local/bin:/usr/local/bin:/bin:/usr/bin:/usr/sbin:${PATH}
-
-help_txt_list=${help_txt_list:="oshmem ompi/mca/coll/hcoll ompi/mca/pml/yall ompi/mca/pml/ucx ompi/mca/spml/ucx"}
-hca_port=${hca_port:=1}
-jenkins_test_build=${jenkins_test_build:="yes"}
-jenkins_test_examples=${jenkins_test_examples:="yes"}
-jenkins_test_oshmem=${jenkins_test_oshmem:="yes"}
-jenkins_test_vader=${jenkins_test_vader:="yes"}
-jenkins_test_check=${jenkins_test_check:="yes"}
-jenkins_test_src_rpm=${jenkins_test_src_rpm:="yes"}
-jenkins_test_help_txt=${jenkins_test_help_txt:="no"}
-jenkins_test_threads=${jenkins_test_threads:="yes"}
-jenkins_test_cov=${jenkins_test_cov:="no"}
-jenkins_test_known_issues=${jenkins_test_known_issues:="no"}
-jenkins_test_all=${jenkins_test_all:="no"}
-jenkins_test_debug=${jenkins_test_debug:="no"}
-jenkins_test_slurm=${jenkins_test_slurm:="no"}
-jenkins_test_comments=${jenkins_test_comments:="no"}
-jenkins_test_ucx=${jenkins_test_ucx:="yes"}
-#jenkins_test_vg=${jenkins_test_vg:="yes"}
-jenkins_test_vg="no"
-#jenkins_test_xrc=${jenkins_test_xrc:="yes"}
-jenkins_test_xrc="no"
-jenkins_test_use_ucx_branch=${jenkins_test_use_ucx_branch:="no"}
-jenkins_test_ucx_branch=${jenkins_test_ucx_branch:="master"}
-jenkins_test_hcoll="no"
+cd "$WORKSPACE"
+export PATH="/hpc/local/bin:/usr/local/bin:/bin:/usr/bin:/usr/sbin:${PATH}"
+
+help_txt_list="${help_txt_list:="oshmem ompi/mca/coll/hcoll ompi/mca/pml/ucx ompi/mca/spml/ucx"}"
+hca_port="${hca_port:=1}"
+ci_test_build=${ci_test_build:="yes"}
+ci_test_examples=${ci_test_examples:="yes"}
+ci_test_oshmem=${ci_test_oshmem:="yes"}
+ci_test_check=${ci_test_check:="yes"}
+ci_test_src_rpm=${ci_test_src_rpm:="no"}
+ci_test_help_txt=${ci_test_help_txt:="no"}
+ci_test_threads=${ci_test_threads:="yes"}
+ci_test_cov=${ci_test_cov:="no"}
+ci_test_known_issues=${ci_test_known_issues:="no"}
+ci_test_debug=${ci_test_debug:="no"}
+ci_test_slurm=${ci_test_slurm:="no"}
+ci_test_comments=${ci_test_comments:="no"}
+ci_test_ucx=${ci_test_ucx:="yes"}
+ci_test_vg="no"
+ci_test_xrc="no"
```

Comment:
xrc flow can be removed. it is deprecated a long ago.

Reply:
Fixed.